nodes/x86_64_ubuntu_16_04_tensorflow.json

```diff
@@ -0,0 +1,18 @@
+{
+  "contact": {
+    "name": "Richard Wei",
+    "email": "rxwei@google.com",
+    "company": "Google"
+  },
+  "node": {
+      "platform": "x86_64",
+      "os_version": "Ubuntu 16.04"
+  },
+  "jobs": [
+    {
+      "display_name": "Swift TensorFlow - Ubuntu 16.04 Linux x86_64 (tensorflow))",
```

Comment:
There's an extra right parenthesis: `(tensorflow))`.
Nitpick: could you change the name to:
```
Swift for TensorFlow - Ubuntu 16.04 Linux x86_64 (tensorflow)
```